### DISCUSSION TITLE
Is this possible?

### DISCUSSION THREAD

**Discussion by amgno** [YimMenu/YimMenu#1860] [Q&A]
Hey I was wondering if it could be possible to:
1. Hide name over your head
2. Display the spoofed player model to local view

Thanks!

**Comment by Alestarov** [comment]
If you are talking about a menu without scripts, then you cannot hide the name and change the model, but you can use a lua script for this. True, such a script will need to be done by yourself, there are no such scripts yet.

**Reply by amgno** [reply]
Not what I mean is, that there is already a spoof player model under > Spoofing, and I just wanted an option to display the spoofed player model locally.
And for the Name hide, I mean when you stand close to someone they can see your name. Do you know a way to do that? Scripts or something like that?

**Reply by Alestarov** [reply]
Unfortunately, there are no such functions in the Yim menu. And I don’t know such scripts, even in other cheats

**Comment by V1RU513N** [comment]
https://github.com/YimMenu/YimMenu/assets/5224766/c5dd433a-31a0-461e-8598-24dc23dc21d5

**Reply by Alestarov** [reply]
He needs the model to render locally
